backbone/src/main/res/drawable/rsb_tremortest1.xml

```diff
@@ -0,0 +1,88 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Nit-pick: Having test in the file name is slightly confusing. If there's another name that makes sense, I'd prefer that.